packages/expo-camera/package.json

```
@@ -2,12 +2,25 @@
   "name": "expo-camera",
   "version": "1.2.0",
   "description": "Expo Camera standalone universal module",
-  "main": "index.js",
+  "main": "build/index.js",
+  "types": "build/index.d.ts",
+  "scripts": {
+   "build": "expo-module build",
+   "clean": "expo-module clean",
+   "test": "expo-module test",
+   "postinstall": "expo-module postinstall",
+   "prepare": "expo-module prepare",
+   "prepublishOnly": "expo-module prepublishOnly",
+   "expo-module": "expo-module"
+  },
   "keywords": [
     "react-native",
     "expo",
     "camera"
   ],
+  "jest": {
+    "preset": "expo-module-scripts"
+  },
```

Could you also please add `repository` key?
```suggestion
  },
  "repository": {
    "type": "git",
    "url": "https://github.com/expo/expo.git"
  },
```